modules/react/banner/lib/Banner.tsx

```diff
@@ -19,15 +19,66 @@ export interface BannerProps extends ExtractProps<typeof Flex, never> {
   children?: React.ReactNode;
 }
 
-const styles: CSSProperties = {
-  cursor: 'pointer',
-  transition: 'background-color 120ms',
-  outline: `${system.space.x1} solid transparent`,
-  '&:focus-visible, &.focus': {
-    outline: `${px2rem(4)} double transparent`,
-    ...focusRing({separation: 2}),
+export const bannerStencil = createStencil({
+  base: {
+    ...system.type.subtext.large,
+    fontWeight: system.fontWeight.medium,
```

Comment:
```suggestion
    ...system.type.subtext.large,
    fontFamily: `${system.fontFamily.default}, Helvetica Neue, Helvetica, Arial, sans-serif`,
    fontWeight: system.fontWeight.medium,
```

Reply:
This will address any other language other than English to fallback to another font-family. 

This will also address the bold issue with the RTL Story.